local-test.js

```diff
@@ -35,6 +35,5 @@ if (!firestoreData) {
     throw new Error('Problem with permission of read and write.\nCheck your firestore permissions')
   }
   await docRef.delete()
+  logger.info('SUCCESS!!! backend setup successful')
```

Comment:
Great! Just one suggestion:
Can we have it in a form language? More formal and professional?

`Success! The backend was successfully set up.`

This is just one suggestion, you can have something of your own as well.

Reply:
Sure!
Changes are done. Thanks